src/Strategy/Node/Pnpm/PnpmLock.hs

```diff
@@ -107,11 +149,17 @@ instance FromJSON PnpmLockfile where
             then Map.insert "." virtualRootWs importers
             else importers
 
-    pure $ PnpmLockfile refinedImporters packages
+    pure $ PnpmLockfile refinedImporters packages rawLockFileVersion
+    where
+      getVersion (TextLike ver) = case (listToMaybe . toString $ ver) of
```

Comment:
We never supported version `<=3` previously, and there is no v7 as of yet.

Reply:
I think we should compare this as a number, and not die on v7 and above by default.
In other words, in the future v7 changes but in a way that doesn't render our parser incompatible, we shouldn't fail to parse it.

Reply:
patched with: https://github.com/fossas/fossa-cli/pull/1320/commits/26204b4dfc6f6372c2c3885bb7c7fea77f1feadd